Makefile

```diff
@@ -37,25 +37,20 @@ build/llvm.BUILT:
 		install-clangQuery \
 		install-clang-apply-replacements \
 		install-lld \
-		install-llc \
-		install-llvm-ar \
 		install-llvm-ranlib \
 		install-llvm-strip \
 		install-llvm-dwarfdump \
 		$(if $(patsubst 8.%,,$(CLANG_VERSION)),install-clang-resource-headers,install-clang-headers) \
-		install-llvm-nm \
-		install-llvm-size \
-		install-llvm-objdump \
-		install-llvm-objcopy \
 		install-ar \
 		install-ranlib \
 		install-strip \
 		install-nm \
 		install-size \
+		install-strings \
 		install-objdump \
 		install-objcopy \
 		install-c++filt \
-		llvm-config
+		install-llvm-config
```

Comment:
Is this needed?  I thought llvm-config was used by people wanting to build program against llvm, not with llvm?   Since we are not shipping libs or headers I'm not sure it makes sense to ship this does it?

Reply:
We already do install libclang (though not libclang-headers, though I've now fixed that), and while I don't know of anyone using it yet, I do have some potential projects that would want to use it.

Reply:
Don't you think we should ship a minimal version of compiler that useable for building?   Keep it small/simple?   Do you think we have used of this SDK who want to build new llvm-based tools, based on what we are shipping here?  Wouldn't such users already a have general purpose llvm-dev installed, or be building from source?

Reply:
Last time I tried a full LLVM install, it installed lots of stuff we don't need.

Would it help if we drop libclang (and headers and llvm-config) for now? Even though I have some ideas in mind for it, I don't urgently need it.

Reply:
I'm happy for you to land this PR in whatever form you see fit.  We can followup with a cleanup of what gets installed later if needed.

Reply:
I decided to go ahead and remove libclang and associated targets. They do add about 50M to the install size, and I don't actually have a use for them right now, so if a use does arise, we can re-evaluate.

Reply:
Awesome